R/driver.R

```diff
@@ -253,13 +262,24 @@ driver <- function(all_data = empty_data(),
         next  # chunk's required inputs are not all available
       }
 
-      if(!quiet) print(chunk)
-
       if(chunk %in% stop_before) {
         chunks_to_run <- character(0)
         break
       }
 
+      if(chunk %in% unfound_inputs$input) {
+        unfound_chunk = unfound_inputs[unfound_inputs$input == chunk, ]
+        chunk_data <- load_csv_files(unfound_chunk$input, unfound_chunk$optional, quiet = TRUE)
```

Comment:
I think it makes sense to shift to this lazy approach. Before, when development was so active and files were changing constantly, it was valuable to load and QC all inputs at the beginning. Now, it's not as important.